addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -299,15 +299,16 @@ options.registry.WebsiteSaleGridLayout = options.Class.extend({
      * @see this.selectClass for params
      */
     setPpg: function (previewMode, widgetValue, params) {
+        const PPG_LIMIT = 10000;
         const ppg = parseInt(widgetValue);
         if (!ppg || ppg < 1) {
             return false;
         }
-        this.ppg = ppg;
+        this.ppg = Math.min(PPG_LIMIT, ppg);
```

Comment:
Nitpicking: I'd just swap those because you usually expect `min`'s parameters to be sorted from low to high.
```suggestion
        this.ppg = Math.min(ppg, PPG_LIMIT);
```